oiosaml/src/main/java/dk/gov/oio/saml/session/AssertionWrapper.java

```diff
@@ -277,4 +258,15 @@ public DateTime getConditionTimeNotBefore() {
 	public DateTime getConditionTimeNotOnOrAfter() {
 		return conditionTimeNotOnOrAfter;
 	}
+
+	public String getSigningCredentialEntityId() {
+		return signingCredentialEntityId;
+	}
+
+	@Override
+	public String toString() {
+		return "AssertionWrapper{" +
```

Comment:
Any reason for this not to be in one line?
And maybe using `String.format()` would make it easier on the eyes.